psalm.xml.dist

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<psalm
+        totallyTyped="true"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="https://getpsalm.org/schema/config"
+        xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <projectFiles>
+        <directory name="bin"/>
+        <directory name="src"/>
+        <directory name="test"/>
+        <ignoreFiles>
+            <directory name="vendor"/>
+            <!-- Temporary suppression of error output -->
```

Comment:
Hmm, these should be in a psalm baseline - not suppressed.

See https://psalm.dev/docs/running_psalm/dealing_with_code_issues/#using-a-baseline-file

The idea is that a baseline prevents introducing even more issues.

Reply:
> Hmm, these should be in a psalm baseline - not suppressed.
> See https://psalm.dev/docs/running_psalm/dealing_with_code_issues/#using-a-baseline-file
> The idea is that a baseline prevents introducing even more issues.

Thanks a lot for the link, I missed something at this point. I'll fix it now.